### PR TITLE
make sure memo change is acted upon

### PR DESCRIPTION
**Pull request by kevodwyer**

### CODE REVIEW

File: assets/apps/calendar/init.js [hunk 1]
```diff
@@ -72,9 +72,10 @@ function buildUI(isCalendarReadonly) {
         'beforeUpdateSchedule': function(e) {
             var schedule = e.schedule;
             var changes = e.changes;
-            if(changes == null) {
-                return;
-            }
+            //do not simply return if no changes. Memo text is not recorded as a change!
+            //if(changes == null) {
```
Comment (ianopolous) [inline assets/apps/calendar/init.js:76]: what was this here for?

Comment (kevodwyer) [reply assets/apps/calendar/init.js:76]: the calendar component stores the changes made in the UI. The Memo field is a custom field we add. It is not part of the calculated changes. It was tempting to short-circuit processing if no change was found. I am making it clear in the code that should not be done.

**Review by ianopolous — APPROVED** [review]

File: assets/apps/calendar/init.js (hunk 1 above)
Comment (ianopolous) [reply assets/apps/calendar/init.js:76]: What the consequence for the user of the short circuit?

Comment (kevodwyer) [reply assets/apps/calendar/init.js:76]: The regression meant the calendar event was not updated when the memo (detailed description) field was updated.

**Pull request merged**